autoendpoint/src/routers/fcm/error.rs

```diff
@@ -91,6 +89,9 @@ impl ReportableError for FcmError {
             FcmError::InvalidAppId(_) | FcmError::NoAppId => {
                 Some("notification.bridge.error.fcm.badappid")
             }
+            FcmError::Upstream { status, .. } if status == "RESOURCE_EXHAUSTED" => {
+                Some("notification.bridge.error.fcm.resource_exhausted")
+            }
```

Comment:
We still have this distinction between how we handle router errors (described in [SYNC-3695](https://mozilla-hub.atlassian.net/browse/SYNC-3695), not sure why we closed this one out?)

The common `handle_error` uses the same metric name for all `RouterErrors`: `notification.bridge.error` w/ tags (most importantly `platform: fcmv1`, `reason: eg_resource_exhausted`). It doesn't handle any more specific kind of errors (like `FcmError`), but even outside of it, I think we should be emitting metrics with the same name and as many as the same tags as possible -- via the `ReportableError` is fine but we probably can't emit `app_id` here without a bunch of other annoying work. That way they're all plotted on the same dashboard panel.

Reply:
This applies to `notification.bridge.error.fcm.badappid` above as well -- we don't plot this one anywhere because of its different naming